napari/_qt/_qapp_model/_tests/test_debug_menu.py

```diff
@@ -0,0 +1,131 @@
+import importlib
+from unittest import mock
+
+import pytest
+
+from napari._app_model import get_app
+from napari._qt._qapp_model._tests.utils import get_submenu_action
+
+
+@pytest.fixture(params=[True, False])
+def perfmon_activation(monkeypatch, request):
+    if request.param:
+        # If perfmon needs to be active add env var and reload modules
+        monkeypatch.setenv('NAPARI_PERFMON', '1')
+        from napari.utils import perf
+
+        importlib.reload(perf._timers)
+        importlib.reload(perf)
+
+        # Check `USE_PERFMON` values correspond to env var state
+        assert perf._timers.USE_PERFMON
+        assert perf.USE_PERFMON
+    else:
+        # If perfmon doesn't need to be active remove env var and reload modules
+        monkeypatch.delenv('NAPARI_PERFMON', raising=False)
+        from napari.utils import perf
+
+        importlib.reload(perf._timers)
+        importlib.reload(perf)
+
+        # Check `USE_PERFMON` values correspond to env var state
+        assert not perf._timers.USE_PERFMON
+        assert not perf.USE_PERFMON
+
+    yield request.param
+
+    # On teardown always try to remove env var and reload `perf` module
+    monkeypatch.delenv('NAPARI_PERFMON', raising=False)
+    from napari.utils import perf
+
+    importlib.reload(perf._timers)
+    importlib.reload(perf)
+
+    # Check `USE_PERFMON` values correspond to env var state
```

Comment:
Is this comment right here or copy pasta?

Reply:
It goes along side the asserts below but maybe I'm being too verbose by commenting the asserts?